packages/dscc-gen/src/files.ts

```diff
@@ -153,25 +168,25 @@ export const cp = async (
   return true;
 };
 
-export const mv = async (
-  fromParts: string[],
-  toDirParts: string[]
-): Promise<boolean> => {
+export const mv = (fromParts: string[], toDirParts: string[]): boolean => {
   const fromPath = path.join(...fromParts);
   const toPath = path.join(...toDirParts);
-  // const result =
-  shelljs.mv('', fromPath, toPath);
-  // if (result.stderr !== null) {
-  //   throw new Error(result.stderr);
-  // }
+  const result = shelljs.mv('', fromPath, toPath);
+  if (!fs.existsSync(toPath)) {
```

Comment:
This should throw whether or not result.stderr is null if the file doesn't exist after moving.

Reply:
yep.